sonic_!amd64_test.go

```diff
@@ -0,0 +1,40 @@
+//go:build !amd64
```

Comment:
old version of go build constraint is also need, to be compatible with old go version